ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/Apps.tsx

```diff
@@ -49,8 +50,21 @@ export function Apps({ isArchived = false }: Props) {
             </div>
           </EmptyAppCard>
         )}
+        {!hasApps && isArchived && (
+          <div className="flex items-center justify-center gap-1.5 rounded-lg bg-slate-500 p-4 text-white">
+            <ExclamationTriangleIcon className="h-5 w-5 text-slate-300" /> No Archived Apps
```

Comment:
The warning triangle is kind of confusing here since it implies "warning", but it's totally fine to have 0 archived apps.

Separate nit: all caps case looks odd here since it's just a sentence fragment and not a title

Reply:
Even though this was copied from the figma file, you're onto something here. I'll leave it to you to decide whether to bring it up or not, but in the meanwhile I'll remove the icon, as I agree with you!